roadmap/implementers-guide/src/types/candidate.md

```diff
@@ -88,57 +85,37 @@ struct CandidateDescriptor {
-
-Unlike the [`GlobalValidationData`](#globalvalidationdata), which only depends on a relay-parent, this is parameterized both by a relay-parent and a choice of one of two options:
-  1. Assume that the candidate pending availability on this para at the onset of the relay-parent is included.
-  1. Assume that the candidate pending availability on this para at the onset of the relay-parent is timed-out.
-
-This choice can also be expressed as a choice of which parent head of the para will be built on - either optimistically on the candidate pending availability or pessimistically on the one that is surely included.
-
-Para validation happens optimistically before the block is authored, so it is not possible to predict with 100% accuracy what will happen in the earlier phase of the [`InclusionInherent`](../runtime/inclusioninherent.md) module where new availability bitfields and availability timeouts are processed. This is what will eventually define whether a candidate can be backed within a specific relay-chain block.
+The validation data also serve the purpose of giving collators a means of ensuring that the candidate they produce will pass the checks done by the relay-chain when backing, and give validators the same understanding.
 
 Design-wise we should maintain two properties about this data structure:
 
-1. The `LocalValidationData` should be relatively lightweight primarly because it is constructed during inclusion for each candidate.
-1. To make contextual execution possible, `LocalValidationData` should be constructable only having access to the latest relay-chain state for the past `k` blocks. That implies
+1. The `ValidationData` should be relatively lightweight primarly because it is constructed during inclusion for each candidate.
+1. To make contextual execution possible, `ValidationData` should be constructable only having access to the latest relay-chain state for the past `k` blocks. That implies
 either that the relay-chain should maintain all the required data accessible or somehow provided indirectly with a header-chain proof and a state proof from there.
 
-> TODO: determine if balance/fees are even needed here.
-> TODO: message queue watermarks (first downward messages, then XCMP channels)
+```rust
+struct ValidationData {
+    persisted: PersistedValidationData,
+    non_persisted: NonPersistedValidationData,
+}
+```
+
+## PersistedValidationData
+
+Validation data that needs to be persisted for secondary checkers.
+    
+These validation data are generally derived from some relay-chain state to form inputs to the validation function, and as such need to be persisted by the availability system to avoid dependence on availability of the relay-chain state. The backing phase of the inclusion pipeline ensures that everything that is included in a valid fork of th relay-chain already adheres to the non-persisted constraints.
```

Comment:
Nit: in the presented version, `ValidationData` briefly describes the general concept and then refers to `NonPersistedValidationData` and `PersistedValidationData`.

I believe that it would be better to do it the opposite way. Have a big description in for the `ValidationData` and put "See ValidationData description for more details" for both `NonPersistedValidationData` and `PersistedValidationData`.

IMO this will give a chance to better structure the description referring both concepts at the same time in one place.

That's however only a nit (not blocking the merge) so feel free to ignore.